docs/usage/faq.md

```diff
@@ -96,42 +100,44 @@ Add a configuration for configuration option `groupName` set to value `"all"`, a
 
 Set configuration option `separateMajorMinor` to `false`.
 
-### Keep using semver ranges, instead of pinning dependencies
+### Keep using SemVer ranges, instead of pinning dependencies
 
 Set configuration option `rangeStrategy` to `"replace"`.
 
 ### Keep lock files (including sub-dependencies) up-to-date, even when `package.json` hasn't changed
 
-This is enabled by default, but its schedule is set to `["before 5am on monday"]`.
-If you want it more frequently, then update the `schedule` field inside the `lockFileMaintenance` object.
+By default, if you enable lock-file maintenance, Renovate will update the lockfile `["before 5am on monday"]`.
+If you want to update the lock file more often, update the `schedule` field inside the `lockFileMaintenance` object.
 
 ### Wait until tests have passed before creating the PR
 
-Set configuration option `prCreation` to `"status-success"`.
+Set the configuration option `prCreation` to `"status-success"`.
 Failing branches will never get a Pull Request created until they eventually pass.
 
+<!-- The above is really, really vague, please help me reword this. -->
```

Comment:
This is quite vague to me:

> Failing branches will never get a Pull Request created until they eventually pass.

Reply:
Can be replaced with "Branches with failing tests will remain in git and continue to get updated if necessary, but no PR will be created until their tests pass".